internal/generator/edwards/template/point.go.tmpl

```diff
@@ -463,26 +470,25 @@ func (p *PointExtended) FromAffine(p1 *PointAffine) *PointExtended {
 // dedicated addition
 // https://hyperelliptic.org/EFD/g1p/auto-twisted-extended-1.html#addition-add-2008-hwcd-4
```

Comment:
change to: https://hyperelliptic.org/EFD/g1p/auto-twisted-extended.html#addition-add-2008-hwcd-2